src/display/api.js

```diff
@@ -304,6 +325,9 @@ function getDocument(src) {
   if (!Number.isInteger(params.maxImageSize)) {
     params.maxImageSize = -1;
   }
+  if (typeof params.standardFontDataWorkerFetch !== "boolean") {
+    params.standardFontDataWorkerFetch = !isNodeJS;
```

Comment:
In the event that the user has provided a *custom* StandardFontDataFactory, it probably doesn't make sense to even attempt to load the font data in the worker-thread. Hence it might not be a bad idea to modify *this* default value as follows?
```suggestion
    params.standardFontDataWorkerFetch = 
      !isNodeJS &&
      params.StandardFontDataFactory instanceof DefaultStandardFontDataFactory;
```